Update resource-limits-quotas-capacity.md (#101187)
Azure/azure-content: articles/load-testing/resource-limits-quotas-capacity.md

| Concurrent engine instances |     5-100 <sup>1</sup>    |     1000    |
| Engine instances per test run | 1-45 <sup>1</sup> |  45   | 

> <sup>1</sup> If you are not already at the maximum limit, you may request an increase. Please note we are not currently able to approve increase requests past our maximum limitations stated above. To request an increase for your default limit, contact Azure Support. Default limits vary by offer category type.

Reviewer: ```suggestion
<sup>1</sup> If you aren't already at the maximum limit, you can request an increase. We aren't currently able to approve increase requests past our maximum limitations stated above. To request an increase for your default limit, contact Azure Support. Default limits vary by offer category type.
```
edits to conform to MS style

| Concurrent test runs | 5-25 <sup>2</sup> |     1000    |
| Test duration | 3 hours |

> <sup>2</sup> If you are not already at the maximum limit, you may request an increase. Please note we are not currently able to approve increase requests past our maximum limitations stated above. To request an increase for your default limit, contact Azure Support. Default limits vary by offer category type.

Reviewer: ```suggestion
<sup>2</sup> If you aren't already at the maximum limit, you can request an increase. We aren't currently able to approve increase requests past our maximum limitations stated above. To request an increase for your default limit, contact Azure Support. Default limits vary by offer category type.
```
